machine/machine_core/constants.py

```diff
@@ -21,6 +21,10 @@
 ATOMIC_IMAGES = ["rhel-atomic"]
 OSTREE_IMAGES = ATOMIC_IMAGES + ["fedora-coreos"]
 
+# Package manager for a given image
+DPKG_IMAGE_PREFIXES = ["debian-", "ubuntu-"]
+RPM_IMAGE_PREFIXES = ["rhel-", "fedora-", "centos-"]
```

Comment:
IMHO this should not be public API. These lists are incomplete -- it's perfectly reasonable to diff the services or openshift image, for example. I think we want to remain able to change the detection implementation, and not third-party projects to use this,